### DISCUSSION TITLE
申请退费

### DISCUSSION THREAD

**Discussion by wikky250** [getlantern/lantern-forum-cn#1006] [Q&A]
![e7ae398b381786c2f7bb3e9208e8d31](https://github.com/user-attachments/assets/88db619f-89b1-4af1-9b31-0cf2b8a431f6)
坐标北京，无法使用。申请退费。
located beijing can't link server，please refund
邮箱：341546586@qq.com

**Comment by wikky250** [comment]
@Mike777Khoo

**Comment by MaxMKMax** [comment]
@wikky250 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Comment by MaxMKMax** [comment]
@wikky250 您好. 已经帮您申请提交退款了.

**Reply by Raymondlee321** [reply]
为什么我的退款还没出来？
78245363
limian321@gmail.com

**Reply by wikky250** [reply]
并未收到退款。

**Reply by wikky250** [reply]
付款邮箱：whmg1234@hotmail.com
